habitat-lab/habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -224,15 +235,20 @@ def _get_obj_samplers(self) -> None:
                         f"Found no object handles for {obj_sampler_info}"
                     )
 
+                if obj_sampler_info["sampler_range"] == "fixed":
+                    num_samples = (
+                        obj_sampler_info["params"]["num_samples"][0],
+                        obj_sampler_info["params"]["num_samples"][1],
+                    )
+                elif obj_sampler_info["sampler_range"] == "dynamic":
```

Comment:
Document what this `dynamic` sampler_range is.

Reply:
Added in `object_sampler.py`